app/src/main/java/org/simple/clinic/recentpatient/RecentPatientScreenItemTypes.kt

```diff
@@ -61,7 +59,7 @@ data class RecentPatientItem(
   data class LastBp(
       val systolic: Int,
       val diastolic: Int,
-      val updatedAtRelativeTimestamp: RelativeTimestamp
+      val updatedAtRelativeTimestamp: org.simple.clinic.summary.RelativeTimestamp
```

Comment:
Statically import this?

Reply:
This has to be explicitly imported as we have two `RelativeTimestamp`s.